filebeat/tests/system/test_crawler.py

```diff
@@ -270,15 +271,19 @@ def test_file_disappear_appear(self):
         """
 
         self.render_config_template(
-            path=os.path.abspath(self.working_dir) + "/log/*",
+            path=os.path.abspath(self.working_dir) + "/log/*.log",
             force_close_files="true",
             scan_frequency="0.1s"
         )
         os.mkdir(self.working_dir + "/log/")
 
         testfile = self.working_dir + "/log/test.log"
+        testfilenew = self.working_dir + "/log/hiddenfile"
         file = open(testfile, 'w')
 
+        # Creates testfile now, to prevent inode reuse
+        open(testfilenew, 'a').close()
+
```

Comment:
Fixing a flaky test. Not related. But it started to pop up frequently in the last 2 days (not sure why). The issue was inode reuse.